applications/Unity.GrantManager/modules/Unity.Payments/src/Unity.Payments.Application/PaymentsApplicationModule.cs

```diff
@@ -60,6 +61,13 @@ public override void ConfigureServices(ServiceConfigurationContext context)
             options.PaymentRequestOptions.ProducerExpression = configuration.GetValue<string>("BackgroundJobs:CasPaymentsReconciliation:ProducerExpression") ?? "";
         });
 
+        Configure<AbpAuditingOptions>(options =>
```

Comment:
This is switching on the Auditing for all entities, then we don't need the [Audited] attribute above and we should modify the code in the core module to have this code instead of just auditing user related entities? 

![image](https://github.com/user-attachments/assets/ac74f2ca-51ea-4cbb-a3cc-d86d10a25a7a)

Reply:
![image](https://github.com/user-attachments/assets/bdbfa8b7-cef4-44f9-be83-efdae0f8cf04)

I am not sure - I think we want auditing on all of these